crates/biome_js_analyze/tests/specs/source/organizeImports/issue-5710.js.snap

```diff
@@ -0,0 +1,10 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: issue-5710.js
+---
+# Input
+```js
```

Comment:
I think we don't need any change set because the new import sorter is not released yet in a stable release.